src/main/java/io/confluent/connect/elasticsearch/DataConverter.java

```diff
@@ -200,12 +207,16 @@ static Schema preProcessSchema(Schema schema) {
             .field(MAP_KEY, preProcessSchema(keySchema))
             .field(MAP_VALUE, preProcessSchema(valueSchema))
             .build());
+        if (schema.isOptional())
```

Comment:
Is it better to `break` and have one single `if (schema.isOptional()) ...` block at the bottom rather than have duplicate code?